README.md

```diff
@@ -131,184 +149,23 @@ darwin-rebuild switch
 
 10. Update `state/darwin-builder.ssh` (in this repo) with the new IP.
 
-[MacStadium]: https://www.macstadium.com
-[Nix]: https://nixos.org/nix
-[NixOps]: https://nixos.org/nixops
-[nix-darwin]: https://github.com/LnL7/nix-darwin
-
-## Deployer
-
-Singleton entity that is used to provision clusters in one-to-many
-relationship. Runs Buildkite to CD clusters on each GitHub push, and to build
-Flatpak bundles, LaTeX documents and HTML documentation as part of CI pipeline.
 
-### Provisioning
-
-```sh
-nix-shell --argstr env production --run 'nixops deploy -d deployer -s state/deployer.nixops'
-```
+--------------------------------------------------------------------------------
 
-## Cluster
 
-Actual Disciplina cluster. WIP.
-
-### Provisioning
-
-Some terminology first:
+# Terminology
 
 * A `deployment` is an entry in a nixops state file.
 * A `cluster` is the set of resources created by running nixops for a given
   deployment.
+* A `deployer` is the singleton entity that is used to provision clusters in
+  one-to-many relationship.
+  Runs Buildkite to CD clusters on each GitHub push, and to build Flatpak bundles,
+  LaTeX documents and HTML documentation as part of CI pipeline.
 
-* A `production` cluster, is the one served at the official DNS name.
-  Deployment access is restricted.
-* A `staging` cluster exists at `*.dscp.serokell.review` and can be updated by
-  most devs, QA, and ops with the `/deploy discplina@<ref> to staging` slack
-  command (not implemented yet).
-* A `testing` cluster belongs to a single dev/qa/ops, and should generally not
-  be expected to be up or reliable, because chances are someone's working on
-  it.
-
-#### IMPORTANT
-
-If you want to avoid hours of building, it's recommended that you set up our binary cache.
-Instructions for this are on the [cachix page](https://disciplina.cachix.org).
-
-You need a patched version of nix. Upstream PR is currently pending. The correct
-version is provided as part of the overlay used in this repo. To install it,
-`cd` into repo base, and:
-
-```sh
-nix-env -f pkgs.nix -iA nix
-```
-
-#### Creating the deployment
-
-To create a new testing deployment, run the following:
-
-```sh
-nixops create deployments/cluster.nix -d disciplina
-nixops set-args --argstr domain yourname.dscp.serokell.review -d disciplina
-nixops deploy -d disciplina
-```
-
-Where `-d disciplina` can be replaced by any name you like, such as
-`discplina-testing` or `disciplina-kawaii`.
-
-To push changes to the cluster, re-run the last command.
-
-When not actively working on your cluster, you should shut it down:
-
-```sh
-nixops stop -d disciplina --confirm
-```
-
-You can start it back up with:
-
-```sh
-nixops start -d disciplina --confirm
-```
-
-To clean up your cluster, which you should do when you're done with it:
-
-```sh
-nixops destroy -d disciplina --confirm
-```
-
-#### Deploying locally
```

Comment:
It might be nice to leave a note that local deployments are temporarily not supported.

Reply:
I would think that not having something mentioned in the docs would imply that it is not supported 🤔

Reply:
If it had never been supported before, sure, but people remember it being supported, so it would be better to mention that it isn't now, rather than just removing the instructions for it.